.github/workflows/python-app.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ "master" ]
+    paths: ['backend/**', '.github/**']
+  pull_request:
+    branches: [ "master" ]
+    paths: ['backend/**', '.github/**']
+
+permissions:
+  contents: read
+
+env:
+  DAY_OF_WEEK: Monday
+  DB_SERVER: localhost
+  DB_PORT: 3306
+  DB_USER: test
+  DB_PASSWORD: test
+  DATABASE: test
```

Comment:
Will these dummy env vars affect testing? We can assign env vars in github actions separately right?

Reply:
https://docs.github.com/en/actions/security-guides/encrypted-secrets should use this instead